package.json

```diff
@@ -66,6 +66,7 @@
   "dependencies": {
     "classnames": "~2.2.6",
     "lodash.sortby": "^4.7.0",
+    "query-string": "^6.2.0",
```

Comment:
ArsArsenal will build a query for you. You just have to return an object from the new `listQuery` option